plugin/src/main/java/io/jenkins/plugins/remotingkafka/KafkaSlaveComputer.java

```diff
@@ -0,0 +1,287 @@
+package io.jenkins.plugins.remotingkafka;
```

Comment:
When you copy-paste the class, make sure to retain the original license. If there is no license header, a repository-wide header should be used.

https://github.com/jenkinsci/jenkins/blob/master/core/src/main/java/hudson/slaves/SlaveComputer.java